modules/apps/html-parser/html-parser-impl/src/main/java/com/liferay/html/parser/internal/activator/HtmlParserBundleActivator.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.html.parser.internal.activator;
+
+import com.liferay.html.parser.internal.CachedLoggerProvider;
+
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+
+/**
+ * @author Guilherme Camacho
+ */
+public class HtmlParserBundleActivator implements BundleActivator {
+
+	@Override
+	public void start(BundleContext bundleContext) throws Exception {
+		CachedLoggerProvider.install();
```

Comment:
@guilhermedcamacho Please move this initialization to HtmlParserImpl. HtmlParserImpl is the only component using jericho code and must be the only one will touch its logger, thus this initialization should be safe to move to HtmlParserImpl#activate